metar/cmd/fetch_metars.go

```diff
@@ -30,16 +30,6 @@ func main() {
 	}
 
 	graph := chart.Chart{
-		XAxis: chart.XAxis{
-			Style: chart.Style{
-				Show: true,
```

Comment:
The `incompatible` API required Show to be set, but the 2.0 released version instead has a `Hidden` boolean so that elements are default enabled.  We forked from this 2.0 API with the `Hidden` field.